flask-api/config/__init__.py

```diff
@@ -34,13 +34,12 @@ def get_environment():
         Returns the appropriate environment configuration.
     '''
     env = os.getenv('ENV_TYPE')
-    
     test = os.getenv('SECRET_1')
     test1 = os.getenv('SECRET_CI_KEY')
     print(test, test1)
 
     if env == 'dev':
         return DevConfig
-    elif env == 'test':
-        return TestConfig 
-    
\ No newline at end of file
+    if env == 'test':
```

Comment:
So for some reason pylint was yelling at me to do this. It was acting like the elif was redundant for some reason though I dont understand how it would be. I can just disable the pylint error for that